src/main/java/offeneBibel/parser/OffeneBibelParser.java

```diff
@@ -503,7 +546,9 @@ public Rule Alternative() {
             push(new ObAstNode(ObAstNode.NodeType.alternative)),
             OneOrMore(FirstOf(
                 ScriptureText(),
+                Sequence("<s>", createOrAppendTextNode(match()), ScriptureText(), "</s>", createOrAppendTextNode(match())),
```

Comment:
Es sollte was durchgestrichenes sein (`<s>` = <s>durchgestrichen</s>).

Lässt sich per etwas wilderem grep über den PageCache auf http://www.offene-bibel.de/wiki/index.php5?title=Markus_8#s22 einschränken :)

[Tipp am Rande, wenn du in Markdown vermeiden willst dass etwas geparst wird, kannst du es entweder mit Backslaschen versehen \\\<s\\\> => \<s\>, oder du kannst es mit (einfachen oder doppelten) Backticks in ne Inline-Code-Section packen: \`\<s\>\` -> `<s>`]